app/mailers/user_mailer.rb

```diff
@@ -240,11 +285,15 @@ def prompter_notification(work_id, collection_id=nil)
     @collection = Collection.find(collection_id) if collection_id
     @work.challenge_claims.each do |claim|
       user = User.find(claim.request_signup.pseud.user.id)
-      mail(
-        :to => user.email,
-        :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] A Response to your Prompt"
-      )
+      I18n.with_locale(Locale.find(user.preference.prefered_locale).iso) do
+        mail(
+          :to => user.email,
+          :subject => "[#{ArchiveConfig.APP_SHORT_NAME}] A Response to your Prompt"
+        )
+       end
```

Comment:
`end` at 293, 7 is not aligned with `I18n.with_locale(Locale.find(user.preference.prefered_locale).iso) do` at 288, 6